### DISCUSSION TITLE
Module 6 and Module 3 Issue with Numpy 2.0 (np.Inf to np.inf)

### DISCUSSION THREAD

**Discussion by chasedamagician** [AccelerationConsortium/ac-microcourses#22] [Hello World] [answered]
Hello, I am currently working on the final module and I have reached an issue that I also encountered in Module 3 but do not how to solve. After the Numpy 2.0 update the function np.Inf was changed to "np.inf" and I suppose one of the libraries has not yet made this change and so I get an AttributeError saying that np.Inf is being used. Is there a way to solve this? Thank you for your time.

**Comment by sgbaird** [comment]
Hi @chasedamagician, thanks for raising this. The incompatibility with `ax-platform` [should be fixed](https://github.com/facebook/Ax/issues/2526) with the next release of `ax-platform`. A temporary workaround (which I've implemented in both the template assignment repo and your specific assignment repo) is to set within `requirements.txt`:
```python
numpy<2.0
```
The same fix was also applied to Module 3.

**Reply by chasedamagician** [reply]
Hello, I am not sure if this issue has solved to np.Inf issue because now when I try to run the code I get the error "No module named numpy". Was this caused by the change to requirments.txt or is this an issue with my code? Thank you for your time.

**Reply by sgbaird** [reply, marked as answer]
I think you resolved by reinstalling, I'm assuming via `pip install -r requirements.txt`. Can you confirm?
